content/docs/1.9.0/v2-data-engine/features/ublk-frontend-support.md

```diff
@@ -0,0 +1,70 @@
+---
+title: UBLK Frontend Support
+weight: 20
+aliases:
+- /spdk/features/ublk-frontend-support.md
+---
+
+Starting with v1.9.0, Longhorn supports UBLK frontend for v2 data engine volume.
+This feature exposes v2 data engine volume as a block device by using [UBLK SPDK framework](https://spdk.io/doc/ublk.html).
+In a very high-spec environment, the UBLK frontend might have better performance than the default NVMe-oF frontend of v2 data engine volume.
+See reference [Longhorn-Performance-Investigation](https://github.com/longhorn/longhorn/wiki/Longhorn-Performance-Investigation).
+However, UBLK frontend is less mature than the default NVMe-oF frontend.
```

Comment:
Consider elaborating slightly on why UBLK is considered `less mature` - maybe specific limitations, missing features? This could help users better understand the trade-offs.

Reply:
Thank you. Fixed